src/apps/compute/videos/index.ts

```
@@ -0,0 +1,29 @@
+      `[/videos/convert-handler] start processing event "${metadata.id}", video "${data.id}"`
+    );
+    video = await convertVideo(data);
+    return res.json(video);
+  } catch (error) {
+    throw AppError('Video conversion failed', {
+      videoId: data.id,
+      error: (error as Error).message,
+    });
+  }
+
+  return video;
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement will return from the function ...

... or this statement will throw an exception beforehand


(lint/correctness/noUnreachable)